pom.xml

```
@@ -111,7 +111,7 @@
         <version.spring>5.1.9.RELEASE</version.spring>
         <version.groovy>2.4.15</version.groovy>
         <version.jackson>2.9.9</version.jackson>
```

If databind is moving to 2.9.10* so should the rest.
```suggestion
        <version.jackson>2.9.10</version.jackson>
```